README.md

```diff
@@ -168,6 +168,7 @@ err := client.Publish(
 * ClientIDs : List of client IDs. Backward compatibility. ([]string - UUID v4 without Hyphens)
 * TargetUserIDs : List of target client IDs. Backward compatibility. ([]string - UUID v4 without Hyphens)
 * TargetNamespace : Target Namespace. Backward compatibility. (string)
+* SourceNamespace : Source Namespace. Usage: if the namespace is a fixed value, such as publisher namespace and event need contain the actual source namespace of this event, then we can use this field. (string  - optional)
```

Comment:
SourceNamespace tracks the trigger namespace, distinct from the event's occurrence namespace.

Reply:
updated